java/README.md

```diff
@@ -0,0 +1,11 @@
+The demo sample uses Spring, gradle and Java 11.
+
+To run with previous versions of Java the Http call handling in KarhooApiClient needs to be changed, but the rest of the code should work as is.
+
+Requests and responses do not have all elements but are a skeleton to enable an initial implementation.
+
+To run the demo:
+
```

Comment:
At java/KarhooDemoApplication.java  Update the credentials

```
String username = "[USERNAME_PLACEHOLDER]";
String password = "[PASSWORD_PLACEHOLDER]";
```

Reply:
Done

Reply:
Done